libraries/AP_Logger/AP_Logger_File.cpp

```diff
@@ -55,10 +55,8 @@ AP_Logger_File::AP_Logger_File(AP_Logger &front,
 }
 
 
-void AP_Logger_File::Init()
+void AP_Logger_File::ensure_log_directory_exists()
```

Comment:
just as a style thing, I would normally put the Init before the "ensure_log_directory_exists()" method.